docs/src/main/sphinx/admin/event-listeners-openlineage.md

```diff
@@ -0,0 +1,241 @@
+  render within produced events.
+- Configure what events to send as detailed
+  in [](openlineage-event-listener-configuration)
+
+(openlineage-event-listener-configuration)=
+
+## Configuration
+
+To configure the OpenLineage event listener, create an event listener properties
+file in `etc` named `openlineage-event-listener.properties` with the following
+contents as an example of minimal required configuration:
+
+```properties
+event-listener.name=openlineage
+openlineage-event-listener.trino.host=<Address of your Trino coordinator>
+openlineage-event-listener.trino.port=<Port of your Trino coordinator>
+```
+
+Add `etc/openlineage-event-listener.properties` to `event-listener.config-files`
+in [](config-properties):
+
+```properties
+event-listener.config-files=etc/openlineage-event-listener.properties,...
+```
+
+:::{list-table} OpenLineage event listener configuration properties
+:widths: 40, 40, 20
+:header-rows: 1
+
+*
+    - Property name
+    - Description
+    - Default
+*
+    - openlineage-event-listener.transport
+    - Type of transport to use when emitting lineage information. 
+      See [](supported-transport-types) for list of available options with
+      descriptions.
+    - `CONSOLE`
+*
+    - openlineage-event-listener.trino.host
+    - Trino hostname. Used to render Job Namespace in OpenLineage. Required.
+    - None.
+*
+    - openlineage-event-listener.trino.port
+    - Trino port. Used to render Job Namespace in OpenLineage. Required.
+    - None.
+*
+    - openlineage-event-listener.trino.includeQueryTypes
+    - Which types of queries should be taken into account when emitting lineage
+      information. List of values split by comma. Each value must be
+      matching `io.trino.spi.resourcegroups.QueryType` enum. Query types not
+      included here are filtered out.
+    - `DELETE,INSERT,MERGE,UPDATE,ALTER_TABLE_EXECUTE`
+*
+    - openlineage-event-listener.openlineage.disabledFacets
+    - Which [](trino-facets) should be not included in final OpenLineage event. 
+      Allowed values: `trino.metadata`, `trino.queryContext`, 
+      `trino.queryStatistics`.
+    - None.
+
+:::
+
+(supported-transport-types)=
+### Supported Transport Types
+
+- `CONSOLE` - sends OpenLineage JSON event to Trino coordinator standard output.
+- `HTTP` - sends OpenLineage JSON event to OpenLineage compatible HTTP endpoint.
+
+:::{list-table} OpenLineage `HTTP` Transport Configuration properties
+:widths: 40, 40, 20
+:header-rows: 1
+
+*
+    - Property name
+    - Description
+    - Default
+*
+    - openlineage-event-listener.openlineage.transport.url
+    - URL of OpenLineage . Required if `HTTP` transport is configured.
+    - None.
+*
+    - openlineage-event-listener.openlineage.transport.endpoint
+    - Custom path for OpenLineage compatible endpoint. If configured, there
+      cannot be any custom path within 
+      `openlineage-event-listener.openlineage.transport.url`.
+    - `/api/v1`.
```

Comment:
these are separete openlineage http transport configuration options so I think it's better to configure them separately - otherwise i'd need to introduce code that splits this in some arbitrary way and passes relevant parts to HttpConfig instance

```
new HttpConfig(new URI(config.getUrl()), config.getEndpoint(), ...)
```